### ISSUE TITLE
Posts that are saved, but not yet published are showing up in public RSS feeds

### ISSUE THREAD

**Issue by natmey** [newamericafoundation/newamerica-cms#1403]
Hi folks,

OTI has a Slack bot that uses an RSS feed to put our posts in the general channel.  Today we noticed a piece get posted that isn't published yet.

The feed we use is:

`https://www.newamerica.org/feed/program/oti/`

The linked piece (that as of writing is still password restricted) is:

```
https://www.newamerica.org/oti/reports/paying-our-privacy-what-online-business-models-should-be-limits/
```

The title "Paying for Our Privacy: What Online Business Models Should Be Off-Limits?"

I would imagine you could replicate this,  but I have not tried to reproduce this at all. Folks in Slack have mentioned that today is not the first time that they have noticed this behavior with reports.

**Comment by nmorduch** [comment]
Thanks @natmey.

@victoriachan I assigned to Kevin for consistency with other issues but, as we've discussed, you can take that just to mean Torchbox.

I'm guessing that the filtering in this line needs to check for whether something is published (which I would think `.live()` would do but BE isn't my thing). And it looks like the other feeds also use similar filtering which would also need to be looked at.
https://github.com/newamericafoundation/newamerica-cms/blob/master/rss_feed/feeds.py#L121

**Comment by nmorduch** [comment]
Changed to using a label for torchbox so that we can use assignment appropriately

**Comment by nmorduch** [comment]
This also happens with:
- Editions of the weekly (has been happening when making a draft of the upcoming edition)
- Reports I think (@mariaelkin can confirm)

I suspect that it's generally happening where we use the API to get lists of pages.

**Comment by mariaelkin** [comment]
In reports, if you publish a report that is password-protected, only the landing page prompts a password. If someone has a direct link to one of the interior page of the report, they can access it without putting in a password.

**Comment by nmorduch** [comment]
I think the reports inside pages is a slightly different problem. I'm actually going to make two new issues for these things.

**Comment by nmorduch** [comment]
Going to leave this open but remove it from the kanban. Consider it a subissue of #1417.

**Comment by nmorduch** [comment]
Closed with #1429
